### DISCUSSION TITLE
how to have creationtimestamp/updatetimestamp features in mongodb with panache

### DISCUSSION THREAD

**Discussion by doogle-oss** [quarkusio/quarkus#31794] [Community]
how to have creationtimestamp/updatetimestamp features in mongodb with panache

in Hibernate ORM we have @CreationTimeStamp and @UpdateTimestamp but with mongodb/panache is it available?? or there is something we need to do at mongo db level.

**Comment by doogle-oss** [comment]
@loicmathieu is there any suggesstion

**Comment by loicmathieu** [comment]
With MongoDB you can use a custom Codec Provider or a Command Listener to build something similar.

**Reply by TheParad0X** [reply]
I would love to see an example in the docs. Coming from a relational DB, this is a very common feature imho 👍

**Reply by haroonsin** [reply]
+1 Pretty useful feature. Would be great if you could advice how we could achieve this?

**Reply by loicmathieu** [reply]
This would be standard MongoDB code, better to ask the MongoDB community of the best way to achieve this using a Codec or a Command Listener.

**Reply by nitinty** [reply]
@debu999 
Did you find any solution for this ?
